### PR TITLE
Plugin extendable logging

### PR DESCRIPTION
**Pull request by alexfurmenkov**

### CODE REVIEW

File: cdisc_rules_engine/services/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
 from cdisc_rules_engine.config import config
-from cdisc_rules_engine.services.logging_service_factory import LoggingServiceFactory
+from cdisc_rules_engine.services.logging import LoggingServiceFactory
 
-logger = LoggingServiceFactory.get_logger(config)
+logger = LoggingServiceFactory(config).get_service()
```
Comment (nhaydel) [inline cdisc_rules_engine/services/__init__.py:4]: We use this logger in the run_validation.py file. In that file we have the option to disable the logger. https://github.com/cdisc-org/cdisc-rules-engine/blob/development/scripts/run_validation.py#L126 Can we add a method to the interface and logger for disabling logging?

Comment (alexfurmenkov) [reply cdisc_rules_engine/services/__init__.py:4]: Updated, implemented the same interface as logging library provides:
image

**Review by nhaydel — APPROVED** [review]

**Comment by nhaydel** [comment]
Approved. If another release is needed, can you merge to main and release from main?

**Comment by alexfurmenkov** [comment]
> Approved. If another release is needed, can you merge to main and release from main?

Sure, will do.

**Pull request merged**